main.ts

```diff
@@ -63,10 +73,10 @@ export default class SmarterMDhotkeys extends Plugin {
 			if (currentClipboardText === relativePath) {
 				// @ts-ignore, basePath not part of API
 				const absolutePath = this.app.vault.adapter.basePath + "/" + relativePath;
-				navigator.clipboard.writeText(absolutePath);
+				await navigator.clipboard.writeText(absolutePath);

```

Comment:
The surrounding function already has async which covers this afaik. If not, please provide a better explanation than "my IDE said to do that", since my IDE didn't say anything :)

Reply:
The function is async, which means that you can await calls inside of it. That means it halts the execution and waits for the result, then it executes the rest. In this case it doesn't matter because nothing after it depends on it, but the writeText method returns a promise, which the code should reflect, I think.
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Reply:
Well, without the await the notice could show before it is written to the clipboard. But that should be very fast, so wouldn't matter in this case.